float64 chunk_x
float64 chunk_z
int16 primary_bitmap
